ParkingZoneApp/Areas/Admin/Controllers/ParkingZoneController.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace ParkingZoneApp.Areas.Admin
 {
-    [Authorize]
```

Comment:
why did you removed that? dont delete write again. dont change parking zone you task is not it